optuna/study.py

```diff
@@ -114,6 +114,20 @@ def get_trials(self, deepcopy: bool = True) -> List[FrozenTrial]:
         For library users, it's recommended to use more handy
         :attr:`~optuna.study.Study.trials` property to get the trials instead.
 
+        Example:
+            .. testcode::
+
+                import optuna
+
+                def objective(trial):
+                    x = trial.suggest_uniform("x", -1, 1)
+                    return x ** 2
+
+                study = optuna.create_study()
+                study.optimize(objective, n_trials=3)
+
+                trials = study.get_trials()
+
```

Comment:
How about adding an assertion like this? It shows that `get_trials` returns all trial clearly.
```suggestion
                trials = study.get_trials()
                assert len(trials) == 3
```

Reply:
Thanks a lot!